static/css/style.css

```
@@ -431,24 +431,38 @@ header {
   transition: all 0.5s ease;
 }
 
-.recruitment-banner {
+.recruitment-banner, .micro-survey-banner {
```

Hide the survey by default with CSS:

```suggestion
.recruitment-banner, .micro-survey-banner {
  display: none;
```